samwise.js

```diff
@@ -1,14 +1,16 @@
+const objectKeys = Object.keys(object);
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.